traffic_ops/traffic_ops_golang/crconfig/edgelocations.go

```diff
@@ -49,20 +49,20 @@ and (st.name = 'REPORTED' or st.name = 'ONLINE' or st.name = 'ADMIN_DOWN')
 
 	for rows.Next() {
 		cachegroup := ""
+		primaryCacheID := ""
```

Comment:
kind of a nitpick, but I think this should be `primaryCacheID := 0` to match the int DB type

Reply:
Addressed.